plugins/modules/inventory_intent.py

```diff
@@ -250,6 +241,10 @@
         description: Preview/Deploy [Preview means the configuration is not pushed to the device. Deploy makes the configuration pushed to the device]
         type: str
         default: "Deploy"
+      clear_mac_address_table:
```

Comment:
clear_mac_address_table: Set this to true if you need to clear the MAC address table for a specific device's interface. It's a boolean type, with a default value of False.

Reply:
updated.